### PR TITLE
Optimize Viewer

### PR DESCRIPTION
**Pull request by tinohager**
Add GraphicField

### CODE REVIEW

File: src/BinaryKits.Zpl.Viewer/ZplElementDrawer.cs [hunk 1]
```diff
@@ -18,6 +18,7 @@ public ZplElementDrawer(IPrinterStorage printerStorage)
                 new GraphicBoxElementDrawer(),
                 new GraphicCircleElementDrawer(),
                 new ImageMoveElementDrawer(),
+                new GraphicFieldElementDrawer(),
```
Comment (YipingRuan) [inline src/BinaryKits.Zpl.Viewer/ZplElementDrawer.cs:21]: Why this guy goes line 21 here ? LOL

**Comment by YipingRuan** [comment]
Would like to learn the many usage of :

`zplCommand.Substring(this.PrinterCommandPrefix.Length)[0];`


Instead of just use string indexer?

**Comment by tinohager** [comment]
Linq Skip and Take?

**Comment by YipingRuan** [comment]
From my understanding, you just want to extract a char from a particular location. How about something like:

`zplCommand[this.PrinterCommandPrefix.Length]`

**Comment by tinohager** [comment]
@YipingRuan Now okay?

**Pull request merged**